packages/amazonq/src/lsp/client.ts

```diff
@@ -183,15 +177,6 @@ export async function startLanguageServer(
         }
     })
 
-    client.onRequest<ShowDocumentResult, Error>(ShowDocumentRequest.method, async (params: ShowDocumentParams) => {
```

Comment:
it might make sense to have this request handled in here instead of inside of `chat/messages.ts` since most of those are just tied to chat?